src/services/saveImagesToUSB.ts

```diff
@@ -0,0 +1,49 @@
+import { IService } from '../types';
+import { readdirSync, existsSync, mkdirSync, rename, statSync, readdir } from 'fs';
+import { USB_WRITE_PATH } from 'config';
+
+export const SaveImagesToUSB: IService = {
+    execute: async () => {
+        try {
+            //Check if USB is connected
+            const usbConnected = existsSync(USB_WRITE_PATH);
+            if (usbConnected) {
+                const directoryPath = USB_WRITE_PATH;
+                const files = readdir(directoryPath, (err, files) => {
+                    files.forEach((file) => {
+                        if (file.endsWith("jpg")) {
+
+                            const formattedDate = new Date().toISOString().split('T')[0];
+
```

Comment:
Sorry for confusion, @HariHive, but I still mean you should take the date from the file name,
I just shared an easy way to get YYYY-MM-DD from it,
your approach with `getDateFromUnicodeTimestamp` was right.

Reply:
updated

Reply:
This is is simply creating a new timestamp of today's date, but we need to accomplish moving the file to the day folder that matches with its timestamp.

Reply:
Feels like Hari didn't commit the fix for this line, 
she initially was using the date from the file name